dapp_runner/runner/runner.py

```diff
@@ -47,26 +54,49 @@ def __init__(self, config: Config, dapp: DappDescriptor):
 
         self.clusters = {}
         self._payloads = {}
+        self._proxies = {}
+        self._networks = {}
         self._tasks = []
         self.data_queue = asyncio.Queue()
         self.state_queue = asyncio.Queue()
 
+    async def _create_networks(self):
+        for name, desc in self.dapp.networks.items():
+            self._networks[name] = await self.golem.create_network(**asdict(desc))
+
     async def _load_payloads(self):
         for name, desc in self.dapp.payloads.items():
             self._payloads[name] = await get_payload(desc)
 
+    async def _start_local_proxy(
+        self, name: str, cluster: Cluster, port_mapping: PortMapping
+    ):
+        port = port_mapping.local_port or get_free_port()
+        proxy = LocalHttpProxy(cluster, port)
+        await proxy.run()
+
+        self._proxies[name] = proxy
+        self.data_queue.put_nowait(
+            {name: {"local_proxy_address": f"http://localhost:{port}"}}
```

Comment:
I think, `"local_proxy_address"` should be a pre-defined constant instead of a hardcoded string

Reply:
Updated in 386d979